src/schemas/21A/schema.js

```diff
@@ -0,0 +1,513 @@
+          },
+          endDate: {
+            $ref: '#/definitions/date',
+          },
+          supervisorName: {
+            type: 'string',
+          },
+          supervisorEmail: {
+            $ref: '#/definitions/email',
+          },
+        },
+      },
+    },
+    financialPlaning: {
+      type: 'string',
+    },
+    homeNursingCare: {
+      type: 'string',
+    },
+    medicalServices: {
+      type: 'string',
+    },
+    consultingService: {
+      type: 'string',
+    },
+    advertisingtoVeterans: {
+      type: 'string',
+    },
+    education: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          name: {
+            type: 'string',
+          },
+          institutionAddress: {
+            type: 'object',
+            properties: {
+              addressType: {
+                type: 'boolean',
+              },
+              line1: {
+                type: 'string',
+              },
+              line2: {
+                type: 'string',
+              },
+              city: {
+                type: 'string',
+              },
+              state: {
+                type: 'string',
+              },
+              postalCode: {
+                type: 'string',
+              },
+              country: {
+                type: 'string',
+                enum: countries.map(x => x.value),
+              },
+            },
+          },
+          startDate: {
+            $ref: '#/definitions/date',
+          },
+          endDate: {
+            $ref: '#/definitions/date',
+          },
+          wasDegreeReceived: {
+            type: 'boolean',
+          },
+          degreeType: {
+            type: 'object',
+            properties: {
+              name: {
+                type: 'string',
+              },
+            },
+          },
+          major: {
+            type: 'string',
+          },
+        },
+      },
+    },
+    jurisdictions: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          name: {
+            type: 'string',
+          },
+          admissionDate: {
+            $ref: '#/definitions/date',
+          },
+          membershipRegistrationNumber: {
+            type: 'string',
+          },
+          admittanceType: {
+            type: 'string',
+          },
+        },
+      },
+    },
+    agencies: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          name: {
+            type: 'string',
+          },
+          admissionDate: {
+            $ref: '#/definitions/date',
+          },
+          membershipRegistrationNumber: {
+            type: 'string',
+          },
+          admittanceType: {
+            type: 'string',
+          },
+        },
+      },
+    },
+    wasImprisoned: {
+      type: 'boolean',
+    },
+    imprisonedExplanation: {
+      type: 'string',
+    },
+    imprisonedDocumentation: {
+      type: 'string',
```

Comment:
I think all documentation fields (file uploads) should have a more complex schema than simply string. 

Our team didn't get around to implementing multi-file uploads, but even single file uploads would have more data:
https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L787

Reply:
@tjheffner - just to be sure, are you suggesting that each of these fields should be like this:
```
imprisonedDocumentation: {
  type: 'object',
  properties: {
    name: {
        type: 'string',
      },
      size: {
        type: 'integer',
      },
      confirmationCode: {
        type: 'string',
      },
    },
  }
}
```

Reply:
Yeah, something along those lines.could likely be simplified by just referencing the common definition directly, e.g.

`imprisonedDocumentation: commonDefinitions.files,`